transformations/diacritic_removal/README.md

```diff
@@ -0,0 +1,16 @@
+# Diacritic Removal 
+This perturbation removes the diacritics on characters in a sentence.
+
+Author: Vikas Raunak (viraunak@microsoft.com)
+
+## What type of a transformation is this?
+This transformation removes the accented characters or diacritics on characters, and replaces them with their non-accented versions.
+
+## What tasks does it intend to benefit?
+This can simplify the dataset to be used for any task, especially which rely on constructing the embeddings from scratch, since often these rare characters with diacritics lead to poorly formed embeddings. 
```

Comment:
There seems to be a word missing in the sentence after `especially`.

Reply:
thanks, made the sentence clear now